subprojects/core/build.gradle.kts

```diff
@@ -107,6 +106,10 @@ dependencies {
     runtimeOnly(libs.groovySql)
     runtimeOnly(libs.groovyTest)
 
+    // The bump to SSHD 2.10.0 causes a global exclusion for `groovy-ant` -> `ant-junit`, so forcing it back in here
+    // TODO investigate why we depend on SSHD as a platform for internal-integ-testing
```

Comment:
💅 Please, create a:chore issue, that way it would be a bit harder to forget.

Reply:
See #27793